src/Stryker.CLI/Stryker.CLI/CLIOptions.cs

```diff
@@ -52,6 +52,15 @@ public static class CLIOptions
             ArgumentShortName = "-p <projectName>",
             ArgumentDescription = @"Used for matching the project references when finding the project to mutate. Example: ""ExampleProject.csproj""",
             JsonKey = "projectName"
-        };        
+        };
+
+        public static readonly CLIOption<int> MaxConcurrentTestRunners = new CLIOption<int>
+        {
+            ArgumentName = "--maxConcurrentTestRunners",
```

Comment:
In PR #129 all argument names are renamed to dash-cased instead of camelCased. It would be nice if we kept consistant by making this argument dash-cased.

Reply:
Sure, will do :)